### DISCUSSION TITLE
Monster movement speed

### DISCUSSION THREAD

**Discussion by seaz5150** [OGSR/OGSR-Engine#296] [Q&A] [answered]
Hello.
 Is there a way in OGSR to determine the current movement speed of a monster (whether it is stationary or not) through a script?

**Comment by xrSimpAI** [comment, marked as answer]
Hi. As far as I know, there is no such way.

**Comment by seaz5150** [comment]
Okay, I wrote a workaround that works pretty much the same way, so it's cool. Thank you for the reply!
